sysutils/fastfetch/Portfile

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem       1.0
+PortGroup        github 1.0
+PortGroup        cmake 1.0
+
+github.setup     fastfetch-cli fastfetch 2.5.0
```

Comment:
please add "github.tarball_from" and set it to "releases" (if upstream provides those) or "tarball" (if not).